src/content/en/tools/lighthouse/https.md

```diff
@@ -0,0 +1,49 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Site is on HTTPS" Lighthouse audit.
+
+{# wf_updated_on: 2016-09-19 #}
+{# wf_published_on: 2016-09-19 #}
+
+# Site is on HTTPS  {: .page-title }
+
+## Why the audit is important {: #why }
+
+All websites should be protected with HTTPS, even ones that don't handle
+sensitive data. HTTPS prevents intruders from tampering with or passively
+listening in on the communications between your site and your users.
+
+HTTPS is also a prerequisite for many new, powerful web platform features, such
+as taking pictures or recording audio.
+
+By definition, an app cannot qualify as a progressive web app if it does not run
+on HTTPS. This is because many core progressive web app technologies, such as
+service workers, require HTTPS.
+
+## How to pass the audit {: #how }
+
+Migrate your site to HTTPS.
+
+Many hosting platforms, such as
+[Firebase](https://firebase.google.com/docs/hosting/) or [GitHub
+Pages](https://pages.github.com/), are secure by default.
+
+If you're running your own servers and need a cheap and easy way to generate
+certficates, check out [Let's Encrypt](https://letsencrypt.org/). For more help
+on enabling HTTPS on your servers, see the following set of docs: [Encrypting
+data in transit](/web/fundamentals/security/encrypt-in-transit/?hl=en)
```

Comment:
Please be careful with your periods and check throughout. I've found quite a few missing.